core/src/main/java/org/apache/calcite/plan/SubstitutionVisitor.java

```diff
@@ -140,7 +145,8 @@
           UnionToUnionUnifyRule.INSTANCE,
           UnionOnCalcsToUnionUnifyRule.INSTANCE,
           IntersectToIntersectUnifyRule.INSTANCE,
-          IntersectOnCalcsToIntersectUnifyRule.INSTANCE);
+          IntersectOnCalcsToIntersectUnifyRule.INSTANCE,
+          SortToSortUnifyRule.INSTANCE);
```

Comment:
`SortOnCalcsToSortUnifyRule` should be implemented.
It's a better way to compare the two rules.

Reply:
Yes, I had planned to support it in the next pr.